packages/angular_devkit/core/node/cli-logger.ts

```diff
@@ -6,7 +6,8 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import { filter } from 'rxjs/operators';
-import { logging, terminal } from '../src';
+import { logging } from '../src';
+import { colors as terminalColors } from './color';
```

Comment:
I am actually inclined to remove colors from here and rely on consumers to pass the colors via the colors parameter when colors are desired.

Reply:
Sounds good.